server/src/main/java/org/elasticsearch/index/IndexingSlowLog.java

```diff
@@ -155,12 +155,12 @@ public static ESLogMessage of(
         private static Map<String, Object> prepareMap(Index index, ParsedDocument doc, long tookInNanos, boolean reformat,
                                                       int maxSourceCharsToLog) {
             Map<String,Object> map = new HashMap<>();
-            map.put("message", index);
-            map.put("took", TimeValue.timeValueNanos(tookInNanos));
-            map.put("took_millis", ""+TimeUnit.NANOSECONDS.toMillis(tookInNanos));
-            map.put("id", doc.id());
+            map.put("elasticsearch.slowlog.message", index);
+            map.put("elasticsearch.slowlog.took", TimeValue.timeValueNanos(tookInNanos).toString());
+            map.put("elasticsearch.slowlog.took_millis", ""+TimeUnit.NANOSECONDS.toMillis(tookInNanos));
```

Comment:
maybe we can use String.valueOf (I missed this bit when working on this last time..)